test/food.spec.js

```diff
@@ -30,6 +30,18 @@ describe('api', () => {
       })
     });
 
+    test('It can get all foods from the database', () =>{
+      return request(app).get('/api/v1/foods').then(response => {
+        expect(response.statusCode).toBe(200)
+        expect(response.body[0].id).toBe(1)
+        expect(response.body[0].name).toBe('Banana')
+        expect(response.body[0].calories).toBe(150)
+        expect(response.body[1].id).toBe(2)
```

Comment:
We might want to change these from specific IDs if we change how our Test Database gets set up in our tests.